testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -910,6 +911,7 @@
 end
 
 When(/^I check "([^"]*)" in the list$/) do |text|
+  raise 'Package name is not set' if text.empty?
```

Comment:
```suggestion
  raise 'The text to check can't be empty' if text.empty?
```

In this particular step definition, we could be using it to mark different kinds of information, it could be a package, but it can also be a channel for example. So, we will need to have a more generic raise error description.